From 17ac5e5396bf506fe4780a97ad1f9f9812c6ee4b Mon Sep 17 00:00:00 2001
From: Dwaine Garden <DwaineGarden@rogers.com>
Date: Mon, 2 Dec 2019 18:47:39 -0500
Subject: [PATCH] Update direwolf.h
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix the following errors during compile.

note: ‘ptrdiff_t’ is defined in header ‘<stddef.h>’; did you forget to ‘#include <stddef.h>’?
---
 src/direwolf.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/direwolf.h b/src/direwolf.h
index efc329b..dff9dea 100644
--- a/src/direwolf.h
+++ b/src/direwolf.h
@@ -37,6 +37,7 @@
 
 #endif
 
+#include <stddef.h>
 
 /*
  * Previously, we could handle only a single audio device.
